azure-pipelines.yml

```diff
@@ -47,11 +47,13 @@ jobs:
    pool:
      vmImage: ubuntu-latest
    steps:
+     - bash: |
+         echo '##vso[task.setvariable variable=Nightly]nightly-'$(curl -s https://rust-lang.github.io/rustup-components-history/x86_64-unknown-linux-gnu/miri)
+       displayName: "Determine latest miri nightly"
+       name: LatestMiri
      - template: install-rust.yml@templates
        parameters:
-         # fix nightly since miri breaks there frequently
-         # should probably be bumped occasionally
-         rust: nightly-2020-03-16
+         rust: $(LatestMiri.Nightly)
```

Comment:
oO WTF what is this.^^ I thought one would just execute the commands I gave as a shellscript. But I also never used Azure CI.

If this works, do you think it is worth putting that into the Miri doc as well?

Reply:
This is the awfulness that is [Azure Pipelines Templates](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops). Unless people are using templates, they shouldn't need this syntax. Here I am using the (very handy) crate-ci/azure-pipelines [install-rust template](https://github.com/crate-ci/azure-pipelines/blob/995c1b8e848994c775ac8349a9801d7dee3a4193/install-rust.yml#L9-L14).